File: leku/src/main/java/com/schibstedspain/leku/geocoder/api/NetworkException.java
package com.schibstedspain.leku.geocoder.api;

public class NetworkException extends RuntimeException {
  public NetworkException() {
  }

  public NetworkException(String message) {
    super(message);
  }

  public NetworkException(String message, Throwable cause) {
    super(message, cause);
  }

  public NetworkException(Throwable cause) {
    super(cause);
  }
}

File: leku/src/main/java/com/schibstedspain/leku/geocoder/api/NetworkException.kt
package com.schibstedspain.leku.geocoder.api

class NetworkException : RuntimeException {
    constructor()

    constructor(message: String) : super(message)

    constructor(message: String, cause: Throwable) : super(message, cause)

    constructor(cause: Throwable) : super(cause)
}
